### ISSUE TITLE
Uncaught Error: Match error: Unknown key in field postSignUpHook

### ISSUE THREAD

**Issue by xvendo** [meteor-useraccounts/core#605]
Here is the code I've been using:
```javascript
var myPostSubmitFunc = function(userId, info) {
  if (userId) {
    Roles.addUsersToRoles(userId, Session.get('loginType'));
  }
};

AccountsTemplates.configure({
  postSignUpHook: myPostSubmitFunc
});
```

**Comment by splendido** [comment]
@xvendo which version of useraccounts are you using?
...`postSignUpHook` is available from `1.13`

**Comment by xvendo** [comment]
Thanks splendido!
I am using 1.12 and will upgrade to 1.13.
